modules/gapi/test/infer/gapi_infer_onnx_test.cpp

```diff
@@ -137,20 +138,27 @@ inline std::vector<int64_t> toORT(const cv::MatSize &sz) {
 }
 
 inline std::vector<const char*> getCharNames(const std::vector<std::string>& names) {
-    std::vector<const char*> out_vec;
-    for (const auto& el : names) {
-        out_vec.push_back(el.data());
-    }
-    return out_vec;
+    std::vector<const char*> out_ptrs;
+    out_ptrs.reserve(names.size());
+    ade::util::transform(names, std::back_inserter(out_ptrs),
+                         [](const std::string& name) { return name.c_str(); });
+    return out_ptrs;
 }
 
 template<typename T>
-void copyToOut(const cv::Mat& in, cv::Mat& out) {
-    const size_t size = std::min(out.total(), in.total());
-    std::copy(in.begin<T>(), in.begin<T>() + size, out.begin<T>());
-    if (size < out.total()) {
-        T* const optr = out.ptr<T>();
-        optr[size] = static_cast<T>(-1); // end data mark
+void copyToOut(const cv::Mat& onnx_out, const T end_mark, cv::Mat& gapi_out) {
+    // This function is part of some remap... function.
+    // You can set graph output size (gapi_out) larger than real out from ONNX
+    // so you have to add something for divide correct data and garbage.
```

Comment:
`to divide`, but divide is usually about math and I'd recommend to use "separate"

Reply:
Changed.